admin/admin-model.php

```diff
@@ -257,15 +257,15 @@ public function update_language( $args ) {
 	}
 
 	/**
-	 * Validates data entered when creating or updating a language
+	 * Validates data entered when creating or updating a language.
 	 *
 	 * @see PLL_Admin_Model::add_language()
 	 *
 	 * @since 0.4
 	 *
-	 * @param array  $args
-	 * @param object $lang optional the language currently updated, the language is created if not set
-	 * @return bool true if success / false if failed
+	 * @param array  $args Parameters of {@see PLL_Admin_Model::add_language() or @see PLL_Admin_Model::update_language()}.
+	 * @param object $lang Optional the language currently updated, the language is created if not set.
+	 * @return WP_Error
```

Comment:
That's strange to have a method that always returns an error. Especially when it's a validation method...

Reply:
Why ? Maybe the function name should be verify_lang and not validate_lang. 
For me it's exactly the same, it's add_language which has to control the result of this function.